tests/fiunchinho/Tests/Silex/Provider/RabbitServiceProviderTest.php

```diff
@@ -87,14 +95,18 @@ public function testAnonymousConsumersAreRegistered()
             ]
         ]);
 
-        $this->assertInstanceOf(Consumer::class, $app['rabbit.anonymous_consumer']['anoymous']);
+        $this->assertInstanceOf(Consumer::class, $container['rabbit.anonymous_consumer']['anoymous']);
     }
 
     public function testMultiplesConsumersAreRegistered()
     {
-        $app = new Application();
+        $container = new Container();
 
-        $app->register(new RabbitServiceProvider(), [
+        $container['debug'] = function() {
+            return true;
```

Comment:
This is a mock for tests, without this tests will not pass because doesn't exists debug function on container.